recipes/umap-learn/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "umap-learn" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "87012d966a61d79cad81703dd1a46c4a48d9a602918899f228b67ac6bc98abc5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and py27 or win32]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.13
+    - numba >=0.35
+    - scikit-learn >=0.19
+    - scipy >=0.18
+
+test:
+  imports:
+    - umap
+  requires:
```

Comment:
For a simple import test, you don't need to require `nose` as far as I can tell. Please remove this if that's the case.

Reply:
How do I run the nose tests themselves? I am a bit of a newbie at the whole conda packaging system, so I'm not entirely sure what the right syntax should be.

Reply:
In fact it doesn't matter; the tests aren't packaged by PyPI (I'll have to fix that in the future) so they aren't available. Removing nose.

Reply:
Typically we don't require you to run the full test suite, but if it doesn't take too long to run and you want to do it, there are a variety of ways. For straight-up unittests, I've done:

```
commands:
    - python -m unittest discover -s tests --verbose
```

You can do something similar with the `nosetests ` command as you do in your own tests. Then you'd want to have `nose` as a `test` requirement. This all assumes you distribute the test modules with the source distribution.